module/behaviour/skills/Getup/src/Getup.cpp

```diff
@@ -49,16 +49,15 @@ namespace module::behaviour::skills {
         : Reactor(std::move(environment))
         , id(size_t(this) * size_t(this) - size_t(this))
         , gettingUp(false)
-        , fallenCheck()
         , FALLEN_ANGLE(0.0f)
         , GETUP_PRIORITY(0.0f)
         , EXECUTION_PRIORITY(0.0f) {
 
         // do a little configurating
         on<Configuration>("Getup.yaml").then([this](const Configuration& file) {
             // encode fallen angle as a cosine so we can compare it directly to the z axis value
-            double fallenAngleConfig = file["FALLEN_ANGLE"].as<double>();
-            FALLEN_ANGLE             = cos(fallenAngleConfig);
+            const float fallenAngleConfig = file["FALLEN_ANGLE"].as<float>();
```

Comment:
Why is this now not a double?

Reply:
other stuff around it was a float, so IIRC it was converting when it was used anyway